src/async_impl/request.rs

```diff
@@ -126,6 +126,49 @@ impl RequestBuilder {
         self
     }
 
+    /// Modify the query string of the URL.
+    ///
+    /// Modifies the URL of this request, adding the parameters provided.
+    /// This method appends and does not overwrite. This means that it can
+    /// be called multiple times and that existing query parameters are not
+    /// overwritten if the same key is used. The key will simply show up
+    /// twice in the query string.
+    /// Calling `.query([("foo", "a"), ("foo", "b")])` gives `"foo=a&boo=b"`.
+    ///
+    /// ```rust
+    /// # use reqwest::Error;
+    /// #
+    /// # fn run() -> Result<(), Error> {
+    /// let client = reqwest::Client::new();
+    /// let res = client.get("http://httpbin.org")
+    ///     .query(&[("lang", "rust")])
+    ///     .send()?;
+    /// # Ok(())
+    /// # }
```

Comment:
These docs are actually showing using the sync API. It's a bit trickier to show docs with the async API, since it requires marking the doctest as needing the `unstable` feature enabled. If it seems complicated (there is an example already, but I don't remember in which file), we can just have no doctest for now.

Reply:
I always forget something. On it.